language/Russian/strings.po

```diff
@@ -5874,22 +5935,23 @@ msgstr "Формат данных"
 
 #: plugins/cheatfind/init.lua:737
 msgid "Slot 1 Value"
-msgstr ""
+msgstr "Значение Слота 1"
 
 #: plugins/cheatfind/init.lua:737
 msgid "Last Slot Value"
-msgstr ""
+msgstr "Значение последнего слота"
 
 #: plugins/cheatfind/init.lua:740
 msgid "Test/Write Poke Value"
-msgstr ""
+msgstr "Тест/Запись указанного значения"
 
 #: plugins/cheatfind/init.lua:747
 msgid ""
 "Use this if you want to poke the Slot 1 value (eg. You started with "
 "something but lost it)"
 msgstr ""
 
+
 #: plugins/cheatfind/init.lua:749
```

Comment:
Please don’t alter the overall formatting of the file like this – it causes trouble when the localisation sources are regenerated.